features/cli/policy.feature

```diff
@@ -463,63 +463,29 @@ Feature: change the policy of user/service account
     Then the step should succeed
     And the output should not match:
       | .*default.*restricted |
-    Given I obtain test data file "authorization/scc/PodSecurityPolicyReview.json"
```

Comment:
@y4sht @redHatGong this case also removes **REDUNDANT** lines of `Given I obtain test data file "authorization/scc/PodSecurityPolicyReview.json"`, it is enough to call one time of this step to obtain the test file.